ui/webui/src/components/users/Accounts.jsx

```diff
@@ -0,0 +1,119 @@
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with This program; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import cockpit from "cockpit";
+import React, { useState, useEffect } from "react";
+
+import {
+    Form,
+    FormGroup,
+    TextInput,
+    Title,
+} from "@patternfly/react-core";
+
+import "./Accounts.scss";
+
+const _ = cockpit.gettext;
+
+const CreateAccount = ({
+    idPrefix,
+}) => {
+    const [fullName, setFullName] = useState();
+    const [userAccount, setUserAccount] = useState();
+    const [password, setPassword] = useState();
+    const [confirmPassword, setConfirmPassword] = useState();
+
+    return (
+        <Form
+          isHorizontal
+          id={idPrefix + "-create-account"}
+        >
+            <Title
+              headingLevel="h2"
+              id={idPrefix + "-create-account-title"}
+            >
+                {_("Create account")}
+            </Title>
+            {_("This account will have administration priviledge with sudo.")}
+            <FormGroup
+              label={_("Full name")}
+              fieldId={idPrefix + "-create-account-full-name"}
+            >
+                <TextInput
+                  id={idPrefix + "-create-account-full-name"}
+                  value={fullName}
+                  onChange={(_event, val) => setFullName(val)}
+                />
+            </FormGroup>
+            <FormGroup
+              label={_("User account")}
+              fieldId={idPrefix + "-create-account-user-account"}
+            >
+                <TextInput
+                  id={idPrefix + "-create-account-user-account"}
+                  value={userAccount}
+                  onChange={(_event, val) => setUserAccount(val)}
+                />
+            </FormGroup>
+            <FormGroup
+              label={_("Password")}
+              fieldId={idPrefix + "-create-account-password"}
+            >
+                <TextInput
+                  id={idPrefix + "-create-account-password"}
+                  value={password}
+                  onChange={(_event, val) => setPassword}
+                />
+            </FormGroup>
+            <FormGroup
+              label={_("Confirm password")}
+              fieldId={idPrefix + "-create-account-confirm-password"}
+            >
+                <TextInput
+                  id={idPrefix + "-create-account-confirm-password"}
+                  value={confirmPassword}
+                  onChange={(_event, val) => setConfirmPassword}
+                />
+            </FormGroup>
+        </Form>
+    );
+};
+
+export const Accounts = ({
+    idPrefix,
+    setIsFormValid,
+}) => {
+    useEffect(() => {
+        setIsFormValid(true);
+    }, [setIsFormValid]);
+
+    return (
+        <>
+            <CreateAccount
```

Comment:
Uneeded wrapper on empty compoent.

Reply:
Yes, it is waiting for the root part. Either I'll keep it or remove when I add root.